testing/coverage/lcov_html.sh

```diff
@@ -63,15 +63,14 @@ while (("$#")); do
             fi
             ;;
         --coveralls)
+            HTML_REPORT=0
             if [ ${#2} -eq 0 ]; then
-                echo "ERROR: Coveralls repo token must be passed with --coveralls argument."
-                echo
-                usage
+                echo "WARN: No coveralls token, running coveralls-lcov --dry-run."
```

Comment:
Can you update the help string for this, I believe the new docstring would be `--coveralls [TOKEN]`, i.e., `TOKEN` is optional.